versioned_docs/version-24.04/installation/inside-centreon.md

```diff
@@ -0,0 +1,88 @@
+---
+id: inside-centreon
+title: Inside Centreon
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Here are the bricks that make Centreon work. To find out on which port each brick communicates, see the [Tables of network flows](https://docs.centreon.com/docs/installation/technical/#tables-of-network-flows).
+
+## Schema of a central server
+
+<Tabs groupId="sync">
+<TabItem value="Central server with poller" label="Central server with poller">
+
+In this example, we have a central server (without any Business modules installed), and a poller that is directly attached to it.
+
+![image](../assets/installation/inside_central_poller.png)
+
+</TabItem>
+</Tabs>
+
+## Elements of a Centreon setup
+
+### Engine
+
+* **Engine**, (the monitoring engine, i.e. the centengine process) is the heart of Centreon: it plans checks, executes them, and [notifies users](https://docs.centreon.com/docs/alerts-notifications/notif-concept.md) if an incident occurs.
+* Checks and notifications are done according to the configuration made by users.
+* Centreon Engine is present on each server in your [architecture](https://docs.centreon.com/docs/installation/architectures.md): [pollers, remote servers and the central server](https://docs.centreon.com/docs/platform.md).
+* To configure Engine, go to **Configuration > Pollers > Engine configuration**. This is for advanced users only.
+
+### cbmod
+
+* Engine’s companion module, **cbmod**, transforms engine events into Broker events, and sends all the data to Broker.
+* Events can be check results, a [downtime](https://docs.centreon.com/docs/alerts-notifications/downtimes.md), an [acknowledgment](https://docs.centreon.com/docs/alerts-notifications/acknowledge.md), a change in the configuration.
+* **cbmod** is a type of [broker](#broker) and can be configured like any Broker using the **Configuration > Pollers > Broker configuration** page.
+
+### Broker
+
+* There are in fact 2 Brokers: 1 **master broker** and 1 **RRD broker**.
+* Only central servers and remote servers contain Brokers.
+* Engine sends data to the master Broker:
+   * The master Broker writes the data in the [MariaDB/MySQL databases](#a-mariadbmysql-database).
+   * It sends a copy of [metrics](https://docs.centreon.com/docs/monitoring/metrics.md) and [statuses](https://docs.centreon.com/docs/alerts-notifications/concepts.md) to the RRD Broker so that it can write them to the [RRD database](#a-rrd-databaserrd-files).
```

Comment:
```suggestion
   * The master Broker sends a copy of [metrics](https://docs.centreon.com/docs/monitoring/metrics.md) and [statuses](https://docs.centreon.com/docs/alerts-notifications/concepts.md) to the RRD Broker so that it can write them to the [RRD database](#a-rrd-databaserrd-files).
```